openedx/core/djangoapps/programs/models.py

```diff
@@ -74,6 +74,11 @@ class ProgramsApiConfig(ConfigurationModel):
         )
     )
 
+    xseries_ad_enabled = models.BooleanField(
+        verbose_name=_("Do we want to show xseries program advertising"),
+        default=True
```

Comment:
I think default False is better - with True, I think it will be enabled immediately on the current config when the migration executes.